src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -462,6 +482,12 @@ private void changeLabels() {
         }
     }
 
+    private void changeMilestoneOfSelectedIssue() {
+        if (getSelectedElement().isPresent()) {
```

Comment:
Hmm, these no-op-if-absent methods aren't so good. If the value was absent to begin with, why call the method? `getSelectedElement().ifPresent(this::changeMilestoneOfSelectedIssue)` is meant for this case. Not sure if the name can be shortened, as it is a bit verbose.